akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -351,6 +351,7 @@ final case class DisabledJavaSerializer(system: ExtendedActorSystem) extends Ser
   }
 
   override def fromBinary(buf: ByteBuffer, manifest: String): AnyRef = {
+    log.warning(LogMarker.Security, "Incoming message is using DisabledJavaSerializer resulting in Message class None! Manifest was: [{}]", manifest)
     this.fromBinary(empty, None)
```

Comment:
However, looking at the `JavaSerializer` it has
```
def includeManifest: Boolean = false
```
i.e. there will never be any manifest for messages that are serialized with the `JavaSerializer` so we should adjust the logging to not mention anything about manifest or Message class.

@2m do you remember what the original issue was, or was it just confusing to see the `None` in the logs?